main.js

```diff
@@ -781,10 +781,26 @@ function sendMessage(text, user, chatId, options) {
         if (typeof user !== 'string' && !(user instanceof Array)) {
             adapter.log.warn('Invalid type of user parameter: ' + typeof user + '. Expected is string or array.');
         }
+	    
+	if (typeof user === 'string') {
+            var user2;
+            user2 = user.replace('\'', '');
+            if (!isNaN(user2)) {
```

Comment:
This check makes no sense ... NaN only happens if you try to cast to a number but it is no number ... the above is a string

Reply:
I found this:

> isNaN(123) //false
> isNaN(-1.23) //false
> isNaN(5-2) //false
> isNaN(0) //false
> isNaN('123') //false
> isNaN('Hello') //true
> isNaN('2005/12/12') //true
> isNaN('') //false
> isNaN(true) //false
> isNaN(undefined) //true
> isNaN('NaN') //true
> isNaN(NaN) //true
> isNaN(0 / 0) //true
> isNaN(null) //false 
> 
> The isNaN() function determines whether a value is an illegal number (Not-a-Number).
> 
> This function returns true if the value equates to NaN. Otherwise it returns false.
> 
> This function is different from the Number specific Number.isNaN() method.
> 
> The global isNaN() function, converts the tested value to a Number, then tests it.
> 
> Number.isNaN() does not convert the values to a Number, and will not return true for any value that is not of the type Number.

So until there is a clean solution, this works for me ;-)
To be honest, i'm not very deep in javascript

Reply:
Ok isNan might behave that way (did't know that) ... and yes if user is a normally a number then I said nothing ;-)